src/clients/identity/token-resolvers/access-token-resolver.ts

```diff
@@ -39,7 +39,7 @@ export class AccessTokenResolver extends TokenResolver<IAccessToken> {
     const entity = await super.verifyToken(token, publicKey);
     let entityWithRoles: IEntityWithRoles | null = null;
 
-    if (options?.permissions?.length || options?.roles?.length) {
+    if (options?.withRolesAndPermissions || options?.permissions?.length || options?.roles?.length) {
```

Comment:
when the flag is up, identity client will always call identity service for permissions, even when they were provided by the token, which is not the intended behavior